autopush/websocket.py

```diff
@@ -1047,17 +1086,23 @@ def _monthly_transition(self):
 
         """
         # Get the current channels for this month
-        _, channels = self.ps.message.all_channels(self.ps.uaid)
+        message = self.db.message_table(self.ps.message_month)
+        _, channels = message.all_channels(self.ps.uaid)
```

Comment:
How come these commands don't need a ddb_resource?

Reply:
`self.db.message_table` automatically uses the resource pool already assigned in `db`. Only stand alone functions require a specific resource handle to be passed in.